### PR TITLE
add test file for pkg/kubectl/cmd/rollout/rollout_pause.go file

### PR DESCRIPTION
**Pull request by WanLinghao**
new:   pkg/kubectl/cmd/rollout/rollout_pause_test.go
        modified: pkg/kubectl/cmd/rollout/BUILD

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by xiangpengzhao** [comment]
/release-note-none
/ok-to-test

please amend your commit message to be more meaningful, also remove the Chinese characters :)

**Comment by WanLinghao** [comment]
@xiangpengzhao thanks for the reply.I am in trouble with the visibility in BUILD file.can you offer any help or clue?

**Comment by WanLinghao** [comment]
@lavalamp @smarterclayton  please take a look :)

**Comment by xiangpengzhao** [comment]
@WanLinghao sorry for the delay. Seems like you have already fixed the BUILD problem :+1: 

In the future, you can try to run `hack/update-bazel.sh` if you change the import of packages in your PR.

**Comment by WanLinghao** [comment]
@kargakis @janetkuo PTAL

**Comment by WanLinghao** [comment]
PTAL,thanks  @dchen1107  @pwittrock

**Comment by WanLinghao** [comment]
@dchen1107 PTAL

File: pkg/kubectl/cmd/rollout/rollout_pause_test.go
Comment (janetkuo) [inline pkg/kubectl/cmd/rollout/rollout_pause_test.go]: Would you also add a switch-case on `req.URL.Path` and `req.Method` like the other unit tests do, and return error when the request is unexpected?

**Comment by WanLinghao** [comment]
@janetkuo  Sorry, I am in trouble with this visibility things, could tell me how to handle this?

**Comment by WanLinghao** [comment]
@janetkuo this patch is ready

**Comment by WanLinghao** [comment]
@janetkuo PTAL Thanks

**Review by janetkuo — APPROVED** [review]

**Comment by janetkuo** [comment]
/assign @monopole 

for review & approval on build visibility related change

**Comment by WanLinghao** [comment]
@soltysh can you plz approve this patch? thanks

**Comment by janetkuo** [comment]
@WanLinghao we need approver from **[build/visible_to/OWNERS](https://github.com/kubernetes/kubernetes/blob/master/build/visible_to/OWNERS)**

File: pkg/kubectl/cmd/rollout/rollout_pause_test.go
Comment (soltysh) [inline pkg/kubectl/cmd/rollout/rollout_pause_test.go]: @janetkuo @monopole @kargakis please do not merge tests that are ONLY verifying the output of the command without verifying the actual change has been applied. This gives false assumption the command is tested, and adds to the overall tech-debt sig-cli already has, and is currently on a path to clean.

Comment (lavalamp) [reply pkg/kubectl/cmd/rollout/rollout_pause_test.go]: Agree, and I thought we didn't make promises about the output format unless you specify some -o option. So this thing ought not be enforced.

Comment (soltysh) [reply pkg/kubectl/cmd/rollout/rollout_pause_test.go]: That is correct.

**Pull request merged**
